docs/research/twitch-websocket-apis-research.md

```
@@ -0,0 +1,1523 @@
+
+## Resources
+
+- **Official Twitch Docs:**
+  - https://dev.twitch.tv/docs/eventsub/
+  - https://dev.twitch.tv/docs/chat/irc/
+  - https://dev.twitch.tv/docs/pubsub/ (migration guide)
+
+- **Community Libraries:**
+  - Twurple: https://twurple.js.org
+  - tmi.js: https://tmijs.com/
+  - Twitch CLI: https://github.com/twitchdev/twitch-cli
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)